rollout/analysis_test.go

```diff
@@ -21,14 +21,16 @@ import (
 )
 
 func analysisTemplate(name string) *v1alpha1.AnalysisTemplate {
+	count := intstr.FromInt(0)
 	return &v1alpha1.AnalysisTemplate{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      name,
 			Namespace: metav1.NamespaceDefault,
 		},
 		Spec: v1alpha1.AnalysisTemplateSpec{
 			Metrics: []v1alpha1.Metric{{
-				Name: "example",
+				Name:  "example",
+				Count: &count,
```

Comment:
This seems to be changing the base template used by the tests. I would expect tests to behave differently because of this change because `count: nil` is different than `count: 0`.  Is this change necessary for some reason?